### ISSUE TITLE
Create *.s19 to Binary converter

### ISSUE THREAD

**Issue by Nahrain1** [amosproj/amos2024ss07-updating-flash-boot-loader#6]
## User story
1. As a Software Developer
2. I want to create a reader for the *.s19 file format
3. So that I can convert it to binary.

## Acceptance criteria
* *.s19 files are parsed an necessary data is successfully extracted  
* Conversion to binary format is accurate (No conversion errors or data loss)
* Error handling is implemented for cases such as malformed *.s19 files or unexpected data formats

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by wiklam** [comment]
What is the expected behaviour for malformed files? Do we want to ignore the malformed entry?

What do you mean by unexpected data format? The extension of a file?
